### PR TITLE
[FIXES MASTER] QoL patch Ocho: It all started with a simple icon fix...

### PR DESCRIPTION
**Pull request by ThePainkiller**
## About The Pull Request

- **FIXES MASTER**: The most important change, due to a compile error from #1678 's indentation error.

- Fixes Ration Can from ERP's not using its proper `opened` and `hot` icons
- Fixes new meat steak sprite never being used.
- Makes the Microwave no longer dense, so that it can be moved atop tables.
- Fixes values on church painkiller chemicals being a massive nerf to their former counterparts, balances their numbers according to litany tier, properly assigns painkiller level to a litany that had a weaker version of what it should have. For clarification, some values went down from 100 painkiller to 20 painkiller effect, others from 30 to 20. This fixes it so that painkiller power of litanies are now somewhat weaker than normal specialized painkiller chems, but strong enough to actually make a difference, while keeping cohesion on litanies who apply them.
TL;DR: Church buff
- Adds proper Bird meat to every type of avian-like creature on the colony, as well as a couple recipes based on chicken meat (so far only chicken roast, chicken steak and fried chicken, feel free to code/sprite more)
- Fixes Meatball recipe creating cooked meatballs instead of raw ones
- Changes burger recipe to take a Patty (new item) instead of a whole slab of meat, or a cutlet, or a meatball. Patties are crafted by using a rolling pin on a raw meat cutlet.
- On the same note, you can no longer craft burgers with cutlets or meatballs in hand, only with patties.
- Adds a stick of butter, sliceable, a component of pies and other recipes that require butter
- Adds pancakes, eggs benedict, eggs and bacon, katsudon, and buttered toast, as well as three recipes with chicken meat
- Adds a bottle of corn oil to the kitchen fridges, comes with 50 units. You'll need it for every recipe that involves frying.
- Changes some recipes' components for cohesion, adds oil and butter where it's needed, as well as salt and pepper and other ingredients (superbite burger now requires bacon, I'll include exact details on the changelog)
- Adds Chicken Steak, Chicken Roast, Fried Poultry, Eggs Benedict, Bacon and Eggs, Katsudon, Pancakes and Buttered Toast
- New sprites for sugar bag, flour pack,  salt shaker, pepper mill, small sugar shaker, fungal meat, stew, bacon, space twinkies, muffin, as well as for the new foods, most ported, some made by yours truly!
- Fixes MRE trash item being slightly off center
- Adds a bag of sugar to the swamp house, better inside illumination, and a padded bed for surgeries
- Vox drop their specific children of poultry meat on butchering now, they contain polystem and quickclot (on @Trilbyspaceclone 's suggestion), and can be used for every chicken recipe
- Hopefully fixes Torches not working properly due to icon update issues
- Comments out trashcart shield as it's uncraftable and unobtainable by virtue of being unable to lift an entire trash cart in your hand to use it on the crafting recipe
- Changes Steel Meat Hook crafting from Plasteel to Steel, reflecting its name (and stats, since it's terrible for everything that isn't throwing it)
- Adds crafting recipe for the spear, makes it a decent throwable weapon

<hr>
Here's a preview of the sprites:

image

Animated stew!

mystew

## Changelog
:cl:
add: Spears can now be crafted, they're robust throwing weapons too
add: Added proper Bird meat to every type of avian-like creature on the colony, recipes can be made with this meat.
add: Added Chicken Steak, Chicken Roast, Fried Poultry, Eggs Benedict, Bacon and Eggs, Katsudon, Pancakes, Butter and Buttered Toast dishes and their recipes, goes as follows:
**Bacon:** 5u Salt + 
**Butter stick:** 5u cream + 5u Salt in a non-microwave recipient. Can be sliced for a **Butter Slice**
**Poultry Roast:** Poultry + 1u Salt + 1u Pepper
**Poultry Steak:** Poultry breast (slice a poultry meat) + 1u salt + 1u Pepper
**Fried Poultry:** 5u Corn Oil + 1u Salt + 1u Pepper + 5u Flour + 1 Poultry Breast
**Eggs Benedict:** 1 Bread Slice + 1 Slice of Butter + 1 Bacon + 1 Boiled Egg + 3u of Egg + 1u Salt + 1u Pepper
**Bacon and Eggs:** 1 Fried Egg + 1 Bacon
**Buttered toast:** Use 1 slice of Butter on a Bread Slice in hand
**Katsudon:** 1 Boiled Rice + 1 Patty (cooked) + 5u Soy Sauce + 3u of Egg
**Pancakes:** 10u Sugar + 5u Milk + 5u Flour + 1 Slice of Butter
del: Comments out trash cart shield as it was impossible to craft since you can't pick up a trash cart to use on the crafting recipe.
tweak: Tweaks A LOT of recipes to now include butter or corn oil in them, as well as other items in their crafting. List goes as follows:
**Fried Eggs** now require 1u of Corn Oil
**Donuts** now require 1u of Corn Oil
**Burgers** can be crafted by hand by slapping one (cooked) Patty unto a Bun, or slapping both into a microwave. They can no longer be crafted by using Meatballs or Cutlets on Buns
**Waffles** are now crafted with 5u Sugar, 5u Milk, 1 Dough Slice and 1 Slice of Butter
**Omelettes** now require 2u of Corn Oil and 1 Slice of Butter
**Popcorn** now requires 2u of Corn Oil and 1 Slice of Butter
**Cookies** now require 1 Slice of Butter
**Big Bite Burger** is now crafted by using a normal Burger, TWO Patties (cooked), and one Fried Egg
**Toasted Sandwich** now requires 1 Slice of Butter
**Grilled Cheese Sandwich** now requires 1 Slice of Butter
**Super Bite Burger** is now crafted with 1 Big Bite Burger, 1 Bun, 2 Patties, 1 cheese wedge and 1 bacon
**Apple Pie** now requires 1 Slice of Butter
**Sausage** is now crafted with 1u Salt, 1u Pepper, a RAW Meatball and a RAW Cutlet
**Plump Helmet Biscuit** now requires 1 Slice of Butter
**Mushroom Soup** is now crafted with 5u Cream, 5u Milk, 1u Salt, 1u Pepper, and 1 Chanterelle mushroom
**Apple Tart** now requires 3u of Egg and 1 Slice of Butter
**Honey Buns** now require 1 Slice of Butter
**French Fries** now require 1u Corn Oil
**Plain Cake** and **ALL** of the cakes children now require **1 WHOLE STICK OF BUTTER** to craft
balance: Rebalanced church chemicals obtained from painkiller litanies so that they better reflect previous stats instead of being a total nerf (church buff)
fix: Fixes Torches (hopefully) and Ration Can icons not using their proper hot and open icon states
soundadd: Changed the sound of the spear to better reflect a stab rather than a slice
imageadd: Added new sprites for Flour Sack, Sugar Sack, Pepper Mill, Salt Shaker, Sugar Shaker, Stew, 
code: Fixes Master due to bad indentation on #1678 's turbine fix
server: Merge this for a fix or do a separate quick hotfix pr as master has been broken for seven hours already
/:cl:

### CODE REVIEW

**Comment by Kazkin** [comment]
I don't like the added lighting to the swamp house, that will probably be reverted by me, the rest is great though. Excellent work.

**Comment by ThePainkiller** [comment]
> I don't like the added lighting to the swamp house, that will probably be reverted by me, the rest is great though. Excellent work.

I just added two more lightbulbs because it physically hurts my eyes to be inside such a dimly lit place for long tbh but sure
